cephfs/admin/fsadmin_test.go

```diff
@@ -0,0 +1,167 @@
+// +build !luminous,!mimic
+
+package admin
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	cachedFSAdmin *FSAdmin
+
+	// set debugTrace to true to use tracing in tests
+	debugTrace = false
+)
+
+func init() {
+	dt := os.Getenv("GO_CEPH_TEST_DEBUG_TRACE")
+	if dt == "yes" || dt == "true" {
```

Comment:
Easier to check if it is set at all? I would set it to `1` or `on` :grin:

Reply:
I was lazy. I can probably just use strconv package.

Reply:
resolved.